docs/tutorials/enterprise_beginner/3_using_existing_blocks.md

```diff
@@ -154,9 +150,9 @@ As you can see, the configuration at this point defines one view for the schedul
 
 See the `data-studio-slots-container` (line 5) and `data-studio-slot` (line 7) attributes? Without them you won't be able to place Content in the slots of the Schedule block, so don't forget about them if you decide to modify the template.
 
-When you look at the template, you can see three blocks, each of which will render the Content items using the `featured` view. As you may remember, so far we only have templates for `full` view for Articles. This means we need to create a `featured` view template for it as well, otherwise we will get an error when trying to add Content to the block.
+When you look at the template, you can see three blocks, each of which will render the Content items using the `featured` view. As you may remember, so far you only have templates for `full` view for Articles. This means you need to create a `featured` view template for it as well, otherwise you will get an error when trying to add Content to the block.
 
-For the app to know which template file to use in such a case, we need to modify the `views.yml` file again. Add the following code to this file, at the same level as the `full` key:
+For the app to know which template file to use in such case, you need to modify the `views.yml` file in `app/config` folder again. Add the following code to this file, on the same level as the `full` key:
```

Comment:
Veto to "in such case" ;) https://www.proz.com/kudoz/English/other/2034947-in_such_a_case_vs_in_such_case.html

Reply:
In such a case
In such case
The former is the most common of the two expressions and is widely used. In such case, however, is mostly retsricted to formal writing. It is very common in law, medicine and the sciences. It wouldn't be a good idea to use this in other registers, for example in fiction or letters to friends. 
from https://english.stackexchange.com/questions/198941/in-such-case-and-in-such-a-case

Reply:
So the question is how "formal" do we consider our texts to be. I'd opt for "in such a case", but I'll leave the choice to you